addons/road-generator/nodes/road_lane_agent.gd

```diff
@@ -153,6 +153,8 @@ func find_nearest_lane(pos = null, distance: float = 50.0) -> RoadLane:
 	var all_lanes:Array = []
 	var groups_checked:Array = [] # Technically, each container could have its own group name
 	var containers = road_manager.get_containers()
+	containers.push_front(road_manager)
```

Comment:
This is the only real fix, other change below is just to get rid of a linter warning/make the typing a bit stricter.